tests/h/search/query_test.py

```diff
@@ -3,188 +3,182 @@
 
 import pytest
 
-from h import search
+from h.search import Search, query
 
 
 class TestTopLevelAnnotationsFilter(object):
 
-    def test_it_filters_out_replies_but_leaves_annotations_in(self, Annotation, search):
-        annotation = Annotation(shared=True)
-        reply = Annotation(references=[annotation.id], shared=True)
+    def test_it_filters_out_replies_but_leaves_annotations_in(self, Annotation, filtered_search):
+        annotation = Annotation()
+        reply = Annotation(references=[annotation.id])
 
-        result = search.run({})
+        result = filtered_search.run({})
 
         assert annotation.id in result.annotation_ids
         assert reply.id not in result.annotation_ids
 
     @pytest.fixture
-    def search(self, pyramid_request):
```

Comment:
Any particular reason for changing the name of the fixture? Usually when we customize a fixture for a particular test case we just override it with one that has the same name.

Reply:
No particular reason-I'll change it to just override the name I supposed.